lib/vdsm/network/link/setup.py

```diff
@@ -23,35 +23,49 @@
 from vdsm.network.link.bridge import Bridge
 
 
+nmstate_options_types = {
+    'mac-ageing-time': int,
+    'group-forward-mask': int,
+    'hash-max': int,
+    'multicast-snooping': bool,
+    'multicast-router': int,
+    'multicast-last-member-count': int,
+    'multicast-last-member-interval': int,
+    'multicast-membership-interval': int,
+    'multicast-querier': bool,
+    'multicast-querier-interval': int,
+    'multicast-query-use-ifaddr': bool,
+    'multicast-query-interval': int,
+    'multicast-query-response-interval': int,
+    'multicast-startup-query-count': int,
+    'multicast-startup-query-interval': int,
+}
+
+
+def _string_to_dict(str, div, eq):
```

Comment:
It's good practice to have helpers after all public functions + for better readiness having the function defined after all callers. That way you can read the file from top to bottom.